lib/committee.js

```diff
@@ -48,6 +48,7 @@ export class Committee {
   addMeasurement (m) {
     assert.strictEqual(m.cid, this.retrievalTask.cid, 'cid must match')
     assert.strictEqual(m.minerId, this.retrievalTask.minerId, 'minerId must match')
+    assert.strictEqual(m.roundId, this.retrievalTask.roundId, 'roundId must match')
```

Comment:
why was this added?

Reply:
I have changed task id structure from `cid::minerId` to `cid::minerId::roundId` hence we're checking round id here.

Reply:
I understand, but what is the motivation?

Reply:
Motivation behind adding the `roundId` to `taskId` was to avoid conflicts when updating `retrieval_timings` values as pkey for the value was day, taskId and minerId. If we'd kept old `taskId` in that structure we would encounter conflicts whenever we'd measure same CID for same storage provider twice in a day. 

I am going to switch table to use int array for storing p50 values for all measurements for a single storage provider and rollback changes I've done around new `taskId`.